core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -57,9 +55,7 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
   <div>
     <background-banner></background-banner>
     <conversation-skin></conversation-skin>
-    {% if not iframed %}
-      <attribution-guide></attribution-guide>
-    {% endif %}
+    <attribution-guide ng-if="!iframed"></attribution-guide>
```

Comment:
Can you show where is iframed defined (in which scope)? (you can add link to the file/line no.)

Reply:
```iframed ``` seems to be defined in pages/Base.js in line 27 [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/Base.js#L27) and this file is extended from base.html

Reply:
Thanks for the link! (LGTM!)